cadquery/occ_impl/shapes.py

```diff
@@ -969,6 +977,47 @@ def innerWires(self):
         
         return [w for w in self.Wires() if not w.isSame(outer)]
 
+    @classmethod
+    def makeNSidedSurface(cls, edges, points, continuity=GeomAbs_C0, Degree=3, NbPtsOnCur=15, NbIter=2, Anisotropie=False, Tol2d=0.00001, Tol3d=0.0001, TolAng=0.01, TolCurv=0.1, MaxDeg=8, MaxSegments=9):
```

Comment:
Same remark a below - maybe good to check the argument naming convention wrt rest of the CQ codebase

Reply:
Done also in ``shape.py`` as explained in above comment https://github.com/CadQuery/cadquery/pull/253#discussion_r363340912